Client/Frontend/Browser/Tab Tray/TabTrayController.swift

```diff
@@ -127,6 +126,10 @@ class TabTrayController: LoadingViewController {
       UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: self, action: nil),
       UIBarButtonItem(customView: tabTrayView.doneButton),
     ]
+    
+    // Place the search bar in the navigation item's title view.
+    searchBarView?.frame = navigationController?.navigationBar.frame ?? .zero
```

Comment:
I don't know if we need this frame assignment in `viewDidLoad` since it seems to layout as expected when view loads, just breaks during rotation when the view is visible. Then I think we can put the `titleView` assignment back in the `do {}` closure above? 🙂

Reply:
Good point let me try and see.